.github/workflows/ci-publish.yml

```diff
@@ -57,7 +57,10 @@ jobs:
       - name: Install from build
         run: |
           python -m pip install --upgrade pip
-          pip install resqpy[tests] --find-links=dist/
+          pip install poetry
+          pip install poetry-dynamic-versioning
+          poetry build
+          # pip install resqpy[tests] --find-links=dist/
```

Comment:
I think this step should be unchanged. Poetry should build the package as a python wheel in a previous step - this  should be pip-installable in the usual way here.